.github/workflows/github-actions-impl-test.yml

```diff
@@ -0,0 +1,100 @@
+name: GitHub Actions Workflow
```

Comment:
If this is for PLT-136 then we need two workflows:

- deploy.yml to deploy code to an environment (in this case, impl)
- e2e-test.yml to run e2e tests on an environment (also on the impl in this case)

The deploy and test on impl should run on push to master and manual dispatch.